charts/spire/charts/spire-server/templates/_helpers.tpl

```diff
@@ -178,3 +178,46 @@ Tornjak specific section
 {{- define "spire-tornjak.backend" -}}
 {{ include "spire-tornjak.fullname" . }}-backend
 {{- end }}
+
+{{/*
+TLS and mTLS Connection Types require tlsSecret to be set before the deployment
+mTLS Connection Types requires userCA.name to be set before the deployment
+*/}}
+{{- define "spire-tornjak.servicename" -}}
+
+{{- if eq .Values.tornjak.config.connectionType "http" -}}
+{{- include "spire-tornjak.backend" . -}}-http
```

Comment:
Do we really need to change the service name when changing the deployment type?

Preferably I would keep the service name the same no matter what deployment type you choose.

In the end this ends up as a dns name.

Reply:
You are right. I kept the various names because earlier we had services and ports for each connection, all active at the same, but since we enforce only one connection, there is no need to use different names.